src/module.ts

```diff
@@ -100,7 +101,9 @@ export default defineNuxtModule({
 
     // Allow extending tailwindcss config by other modules
     // @ts-ignore
-    await nuxt.callHook('tailwindcss:config', tailwindConfig)
+    const _tailwindConfig = await nuxt.callHook('tailwindcss:config', tailwindConfig)
```

Comment:
I am not sure regarding this since it is an object at first, so using hooks can easily mutate the object keys directly, no need to return anything.